app/templates/src/main/java/package/config/_DatabaseConfiguration.java

```diff
@@ -114,9 +116,9 @@ public SpringLiquibase liquibase(DataSource dataSource) {
         SpringLiquibase liquibase = new SpringLiquibase();
         liquibase.setDataSource(dataSource);
         liquibase.setChangeLog("classpath:config/liquibase/master.xml");
-        liquibase.setContexts("development, production");
+        liquibase.setContexts(liquiBasePropertyResolver.getProperty("context", "development, production"));
```

Comment:
@jdubois I put by default your old configuration so I don't change the old  behavior. It's good for you?